compiler/noirc_evaluator/src/ssa/plonky2_gen/asm_writer.rs

```diff
@@ -181,11 +207,14 @@ pub struct AsmWriter {
     last_line_number_begin: usize,
     last_line_number_end: usize,
     file_map: FileMap,
+    // information, stored, for use with the 'nargo trace --trace-plonky2' feature
+    pub debug_trace_list: Option<DebugTraceList>,
+    prev_dsp: Option<DebugTraceSourcePoint>,
```

Comment:
Naming.

prev_source_point?
prev_point?
prev_debug_trace_source_point? (hello, java!)
prev_dtsp?

It feels like we could improve `dsp`.

Reply:
prev_source_point: https://github.com/blocksense-network/noir/pull/96/commits/9db2d2f1c7d601e01a0557b14d362c02a9e5855d